test/functional/autoread/focus_spec.lua

```diff
@@ -0,0 +1,64 @@
+local helpers = require('test.functional.helpers')(after_each)
+local thelpers = require('test.functional.terminal.helpers')
+local clear = helpers.clear
+local retry = helpers.retry
+local nvim_prog = helpers.nvim_prog
+local feed_command = helpers.feed_command
+local feed_data = thelpers.feed_data
+
+if helpers.pending_win32(pending) then return end
+
+describe('autoread TUI FocusGained/FocusLost', function()
+  local screen
+
+  before_each(function()
+    clear()
+    screen = thelpers.screen_setup(0, '["'..nvim_prog
```

Comment:
Can't you just use `screen = thelpers.screen_setup()` here?

Reply:
I don't think I can, since we open a child neovim instance and trigger the `FocusGained` and `FocusLost` event in this child instance, and see if the file was reloaded. The defualt command for `screen_setup` is `nvim_dir/tty-test`. I wasn't too sure of whether this is what was required and also saw that this was what was already being used in [tui_spec.lua#L906](https://github.com/neovim/neovim/blob/master/test/functional/terminal/tui_spec.lua#L906) so decided using this.